sdk/storage/azure-storage-blob/tests/test_common_blob_async.py

```diff
@@ -1041,7 +1033,7 @@ def callback(response):
 
 
     # @GlobalStorageAccountPreparer()
-    # @AsyncBlobTestCase.await_prepared_test
+    # @AsyncStorageTestCase.await_prepared_test
     # TODO: external copy was supported since 2019-02-02
     # async def test_copy_blob_with_external_blob_fails(self):
     #     # Arrange
```

Comment:
We shouldn't leave this commented out. Either the test can be removed because the service has changed, or it should be updated.